template-parts/partners-grid.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * Climate Hub partners page grid template
+ *
+ * @package climatehub
+ */
+
+if( get_field('enable_partners_grid') ): ?>
```

Comment:
Instead of hardcoding every logo, you can loop through all the logos and generate the grid. Something like this would work:
```php
if( get_field('enable_partners_grid') ): ?>
  <div class="container">
    <div class="row">
      <?php
        $size = 'full';
        $numRows = 3;
        $numLogos = 5;
        for($row=1; $row<=$numRows; $row++) {
          for($logo=1; $logo<=$numLogos; $logo++) {
            if ($image = get_field('row_'.$row)['logo_'.$logo]) { ?>
              <div class="col-12 col-sm-6 col-md-4 col-lg-3 custom-col-xl-2-4 mb-5">
                <div class="partner-logo-container m-auto">
                  <a href="<?php echo wp_get_attachment_caption($image) ?>">
                    <div class="partner-logo" style="background-image: url('<?php echo wp_get_attachment_image_url($image, $size) ?>')"></div>
                  </a>
                </div>
              </div>
            <?php 
            }
          }
        }
      ?>
    </div>
  </div>
<?php endif ?>
```
You will also need some css to get it working:
```css
@media (min-width: 1200px) {
  .custom-col-xl-2-4 {
    flex: 0 0 20% !important;
    max-width: 20% !important;
  }
}

.partner-logo-container {
  max-width: 15rem;
}

.partner-logo {
  padding-top: 100%;
  background-origin: border-box;
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
}
```